### DISCUSSION TITLE
Table View for Media Collection no longer shows the thumbnail gallery

### DISCUSSION THREAD

**Discussion by lilnotes** [payloadcms/payload#2540] [Q&A] [answered]
Current version: `"payload": "1.6.32"`
The previous version where it worked: `"payload": "1.6.22",`

also I use
`"@payloadcms/plugin-cloud-storage": "^1.0.12",`

After I've updated the payload version, all my media collections look like Table Views (columns with non-image data instead of thumbnails).
And it works fine for these collections that contain some documents, but I prefer to see thumbnails if the collection only contains image files.

Is it possible to return them back? Or to configure it using some "admin" property?

UPD: I found that if I select "File name" column I can see previews 🙈 (but it's so not obvious)

**Comment by DanRibbens** [comment, marked as answer]
Hey @lilnotes 
This was a UX decision we made when implementing bulk operations. Several others have asked for different list views in other discussions and we think it is a great idea!

If you don't want to wait for us to get to it you can always make a custom List component and assign it to admin.components.List in your upload enabled collections. It might be a bit of work but you could look at the old UI component as your guide https://github.com/payloadcms/payload/tree/v1.6.23/src/admin/components/elements/UploadGallery
Which gets used in: src/admin/components/views/collections/List/Default.tsx

**Comment by 58bits** [comment]
Hi @lilnotes @DanRibbens I've just had a go at this here...
https://github.com/infonomic/payload-gallery-view
